src/scripts/main.js

```diff
@@ -1,5 +1,30 @@
 'use strict';
 
+const spider = document.getElementsByClassName('spider')[0];
+const wall = document.getElementsByClassName('wall')[0];
+
 document.addEventListener('click', (e) => {
-  // write code here
+  if (e.target.closest('.spider')) {
+    return;
+  }
+
+  if (e.target.closest('.wall')) {
+    const spiderCentered = spider.clientWidth / 2;
+
+    let targetX = e.offsetX - spiderCentered;
+    let targetY = e.offsetY - spiderCentered;
```

Comment:
The calculation of 'targetX' and 'targetY' assumes that the click event's offset properties are available and meaningful. Ensure that the event is triggered on an element where these properties are defined, or handle cases where they might not be.